db/install.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="blocks/xp/db" VERSION="20160224" COMMENT="XMLDB file for Moodle blocks/xp"
+<XMLDB PATH="blocks/xp/db" VERSION="20170214" COMMENT="XMLDB file for Moodle blocks/xp"
```

Comment:
Have you used the `XMLDB Editor` tool under `Site administration > Development`. The latter takes care of maintaining that XML file, it also providers the PHP upgrade code.